testing/README.md

```diff
@@ -2,26 +2,27 @@
 
 ## Philosophy
 * Test code is a first-class part of the deliverable, not an optional nice-to-have.
-* When scopeing / estimating work: always factor in testing. 
-* Untestable code = bad code.
+* When scoping / estimating work: always factor in testing. 
 * Insufficent test coverage is better than no test coverage.
+* Prefer [Use Case Coverage over Code Coverage](https://kentcdodds.com/blog/how-to-know-what-to-test) (no need for 100% code coverage).
```

Comment:
I strongly believe 100% code coverage should not be a goal. Related to that, I was introduced to the idea of "Use Case Coverage" over "Code Coverage" and have found it to be a much more fruitful way to approach writing tests. I'd like to suggest we also embrace this as a team but would like to know how others feel. 🙂